Dispatchfile

```diff
@@ -27,21 +29,16 @@ task("goimports-check", inputs=["kudo-cassandra-operator"], steps=[k8s.corev1.Co
   )
 )])
 
-
```

Comment:
Not your change> but I observe that `goimports` step is requesting for
> 7.5Gi
of memory.